docs/UserGuide.md

```diff
@@ -1,186 +1,85 @@
-# AddressBook Level 4 - User Guide
+# User Guide
 
-By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbsp;&nbsp; Licence: `MIT`
+This product is not meant for end-users and therefore there is no user-friendly installer. 
+Please refer to the [Setting up](DeveloperGuide.md#setting-up) section to learn how to set up the project.
 
----
+## Starting the program
```

Comment:
You are going to provided `.jar` file in the end. We run your application through `.jar` file rather than use eclipse.